tutorials/profiler/tokenizer.py

```diff
@@ -0,0 +1,112 @@
+import os
```

Comment:
I agree with @msaroufim we should not duplicate this code (and model.py). We already copied them from gpt-fast so we should only have 1 version in torchao. Can you merge these with the existing ones under `torchao/_models`?

Reply:
@jeromeku I think this is the only controversial thing about this PR, if we could reduce code duplication this would make merging this a no brainer

We're planning a release on Aug 8, do you think you'll have time to land your changes before then?